### PR TITLE
Prettify Profile Page

### PR DESCRIPTION
**Pull request by blrubenstein**
Per this:
https://projects.invisionapp.com/d/main#/console/12820961/286774501/preview

Final changes are going to come with some API additions, so that is coming next.

### CODE REVIEW

File: ui/src/app/views/profile-page/component.ts [hunk 1]
```diff
@@ -47,16 +71,62 @@ export class ProfilePageComponent implements OnInit {
...
+      completedTasks += 1;
+    }
+    if (this.profile.termsOfServiceCompletionTime !== null) {
+      completedTasks += 1;
+    }
+    return completedTasks;
+  }
+
+  public get completedTasksAsPercentage() {
+    return this.completedTasks * 25;
```
Comment (johnstill) [inline ui/src/app/views/profile-page/component.ts]: I would stick `4` in a `const` somewhere and then use the actual percentage formula here - 25 makes no sense if the number of tasks changes (or if you just don't know what the total number _should_ be).  Same below (line 106).  Then you can avoid having `25` and `4` hanging out as magic numbers.

File: ui/src/app/views/profile-page/component.ts [hunk 2]
```diff
@@ -47,16 +71,62 @@ export class ProfilePageComponent implements OnInit {
       });
   }
 
+  public get completedTasks() {
+    if (this.profile === undefined) {
+      return 0;
```
Comment (johnstill) [inline ui/src/app/views/profile-page/component.ts]: You can put the initializer above this and `return completedTasks;` here - removes another magic number (not _very_ magic b/c the definition is right below, but still).

File: ui/src/app/views/profile-page/component.ts [hunk 3]
```diff
@@ -1,33 +1,57 @@
 import {Component, OnInit} from '@angular/core';
 import {ActivatedRoute, Router} from '@angular/router';
 import {ErrorHandlingService} from 'app/services/error-handling.service';
+import {SignInService} from 'app/services/sign-in.service';
 
-import {ErrorResponse, Profile, ProfileService} from 'generated';
+import {BlockscoreIdVerificationStatus,
```
Comment (johnstill) [inline ui/src/app/views/profile-page/component.ts]: NIT: drop and dedent this please. e.g.

```
import {
  symbolA,
  symbolB,
  etc,
} from 'wherever';
```

Trailing commas are ok too.

File: ui/src/app/views/profile-page/component.html [hunk 4]
```diff
@@ -19,110 +19,109 @@ <h2>Profile</h2>
...
+      </div>
+      <div>
+        <button class="btn btn-secondary" (click)="submitTermsOfService()"
+            [disabled]="email===null || this.profile.termsOfServiceCompletionTime !== null">
+          Submit Terms of Service
+        </button>
+      </div>
+      <div>
+        <button class="btn btn-secondary" (click)="completeEthicsTraining()"
+            [disabled]="email===null || this.profile.ethicsTrainingCompletionTime !== null">
```
Comment (johnstill) [inline ui/src/app/views/profile-page/component.html]: NIT: use spaces around operators (`===`) please.  See also lines 109 and 121 (at least).

**Comment by blrubenstein** [comment]
I took some of the inline styles, where there were a lot, and made that into a class, but I think some of the single value styles I can leave as inline, because it may actually be more descriptive that way?

**Pull request merged**